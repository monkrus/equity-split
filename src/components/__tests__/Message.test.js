import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import MessagePopover from '../Message'

describe('MessagePopover', () => {
  const text = 'test';
  const component = shallow(
    <MessagePopover placement='bottom' message='test'/>
  );

  //Generic Snapshot Test
  it('renders and matches snapshot', () => {
    const component = renderer.create(
      <MessagePopover id='t1' placement='bottom' message='test'/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  //Content Test
  it('contains test text', () => {
    expect(component.text()).toContain(text)
  });
})
