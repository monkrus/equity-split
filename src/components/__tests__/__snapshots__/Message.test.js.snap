exports[`MessagePopover renders and matches snapshot 1`] = `
<div
  className="popover bottom"
  id="t1"
  role="tooltip"
  style={
    Object {
      "display": "block",
      "left": undefined,
      "top": undefined,
    }
  }>
  <div
    className="arrow"
    style={
      Object {
        "left": undefined,
        "top": undefined,
      }
    } />
  <div
    className="popover-content">
    test
  </div>
</div>
`;
